hail/python/hailtop/hailctl/dataproc/resources/init_notebook.py

```diff
@@ -39,14 +39,15 @@ def mkdir_if_not_exists(path):
     pip_pkgs = [
         'setuptools',
         'mkl<2020',
-        'ipywidgets<8',
-        'nbconvert<6',
-        'notebook<6',
-        'qtconsole<5',
         'lxml<5',
-        'google-cloud==0.32.0',
+        'google-cloud==0.32.*',
```

Comment:
this is somewhat unnecessary given that jgscm has its own google cloud dependency, but sure

Reply:
Why do we include google-cloud as a dep?